main.py

```diff
@@ -65,9 +66,16 @@ def change_read():
     book.read = new_read 
     book.save()
     
+def delete_book():
+    try:
+        search_id = ui.ask_question('Enter Book ID:')
+        match_id = store.get_book_by_id(search_id)
+        store._delete_book(match_id)
```

Comment:
Don't call the store method directly - see note here. Can you ask the book object to delete itself? https://github.com/DashKazak/readinglist/blob/e7b462ef3986b08f26a62c290fb0ae277cc53ae3/bookstore.py#L82-L91